pxl/compress.py

```diff
@@ -47,3 +49,35 @@ def compress_image(local_filename: Path) -> Dict[state.Size, Path]:
             image_paths[size_to_generate] = scaled_path
 
     return image_paths
+
+
+def orient_exif(image: Any) -> Any:
+    """
+    Rotate the image according to EXIF metadata.
+    """
+    exif_data = image._getexif()
+
+    orientation_tag = 274
```

Comment:
Can you add a comment (maybe with link to spec) to explain the magic number here?

Reply:
The spec link isn't too easy (it's a PDF): https://www.exif.org/Exif2-2.PDF search for "Orientation"

Reply:
But I documented this